backend-zs/.env

```diff
@@ -0,0 +1,9 @@
+API_PORT=3001
```

Comment:
We already have all the variables in env.sample, so we either not add this one or remove `env.sample`

Reply:
The file was missing from gitignore. I added it now.